RELEASING.md

```diff
@@ -88,6 +88,115 @@ Then you can work on the PR. Once done, push your changes back to the PR. If eve
 
 There are some known issues that commonly need to be addressed by the BOM release manager.
 
+##### Handling Rollbacks
+
+Sometimes a dependency update that was merged and released causes compatibility issues or test failures that are discovered after the BOM release. When this happens, a rollback process is needed to temporarily revert the problematic change while the issue is investigated and resolved.
+
+###### When to Consider a Rollback
+
+Consider rolling back a dependency update when:
+
+- The updated dependency causes test failures in downstream plugins
+- The update introduces breaking changes that affect plugin compatibility
+- Critical functionality is broken after the dependency update
+- The issue cannot be quickly resolved and is blocking the release process
+
+###### Rollback Process
+
+When a problematic dependency update needs to be rolled back, follow these steps:
+
+1. Notify the Author
+
+**Notify the author of the problematic change as close as possible to the actual problem area.** Leave a comment on the original PR or commit that introduced the issue, explaining the specific problems encountered.
+
+**Example:**
+```
+Looks like these changes are causing issues with doing a BOM release this week. For reference:
+
+tl;dr...two plugins that passed on the Tuesday weekly-test (foo-plugin and bar-plugin) are now failing.
+
+For the moment so I can get the BOM release out today, I'm going to revert the PR that put in github-branch-source 1844.v4a_9883d49126.
+
+The local BOM commands I'm running that are failing are:
+- `LINE=weekly PLUGINS=bar-plugin bash local-test.sh`
+- `LINE=weekly PLUGINS=foo-plugin bash local-test.sh`
+
+As noted, when I locally go back to github-branch-source 1834.v857721ea_74c6 both tests are passing.
+```
+
+**Reference:** See [this example notification](https://github.com/jenkinsci/github-branch-source-plugin/pull/822#issuecomment-3238052920) to the plugin author.
+
+2. Revert the Change
+
+Create a revert commit that rolls back the problematic dependency update. In your commit message, include a link to the notification you sent to the author.
```

Comment:
No particular opinion about the commit message, but I find it helpful to include this information in the PR description additionally/instead. Makes for more legible hover tips and better backlinks.